src/components/TalkDetail.astro

```diff
@@ -23,15 +25,26 @@ export const talkDetailFragment = graphql(`
 				title
 			}
 		}
+		speakersCollection(limit: 3) {
```

Comment:
Limit query complexity, assuming we'll never have more than 3 speakers per talk.